app/views/concerns/entity_displaying_view.rb

```diff
@@ -217,7 +206,7 @@ def entity_death
 
   def entity_date(date)
     # Just grab the first date in the array if present.
-    date && date.is_a?(Array) && date.length && date.first.is_a?(String) ? date.first : nil
+    date && date.is_a?(Array) && date.length && date.first.is_a?(String) ? date_eras_gregorian(date.first) : nil
```

Comment:
I was told by Hugo that we should always use the "longest" date in the array. See: https://app.assembla.com/spaces/europeana-npc/tickets/realtime_list?ticket=2335